Code/panel.js

```diff
@@ -307,6 +307,21 @@ define([
             return false;
         },
 
+        /**
+         * updates button in the panel for tooltip.
+         * @function module:wcPanel#updateButton
+         * @param {Array} buttons - List of buttons to update.
+         * @returns {Boolean} - Success or failure.
+         */
+        updateButton: function (buttons) {
```

Comment:
Why are you passing the complete list to `updateButton` and replacing the button list ? This is function is not re-usable at all. It should allow changing data for a single button.